elasticsearch/pom.xml

```diff
@@ -136,12 +133,17 @@
                         </goals>
                         <configuration>
                             <archive>
-                                <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                                <manifestEntries>
+                                    <Automatic-Module-Name>org.hibernate.search.backend.elasticsearch</Automatic-Module-Name>
```

Comment:
You should also add a module name to the `hibernate-search-elasticsearch-aws` module. I would go with `org.hibernate.search.backend.elasticsearch.aws`.